src/App.jsx

```diff
@@ -36,36 +40,70 @@ function App() {
     setMoves((currentMoves) => currentMoves + 1);
   };
 
+  let rgb = [...rgbStart];
+  if (moves <= 60) {
+    rgb[0] += 3 * moves;
+    rgb[1] -= 3 * moves;
+    rgb[2] -= moves;
+  } else {
+    rgb = [...rgbEnd];
+  }
```

Comment:
This is an interesting way to handle the color transitions! What you're effectively doing here is a bit of a manual [linear interpolation](https://en.wikipedia.org/wiki/Linear_interpolation). Check out the section on "Programming language support".

```ts
function lerp(start: number, end: number, t: number) {
  return (1 - t) * start + t * end;
}
```

The above is for a single number, but you could easily extend it to lerp a 3-element array or lerp the r, g, and b values individually.

The approach I took for this was to use a different color representation: HSL which can be a little easier to manipulate logically.